x11-misc/libinput-gestures/libinput-gestures-2.38.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{3_4,3_5,3_6} )
+
+inherit gnome2-utils python-single-r1
+
+DESCRIPTION="Actions gestures on your touchpad using libinput"
+HOMEPAGE="https://github.com/bulletmark/libinput-gestures"
+if [[ ${PV} == *9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/bulletmark/${PN}.git"
+else
+	SRC_URI="https://github.com/bulletmark/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="test"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="${PYTHON_DEPS}
+	>=dev-libs/libinput-1.8.0
+	x11-misc/xdotool
+	x11-misc/wmctrl"
+DEPEND=">=dev-libs/libinput-1.8.0
+	dev-util/desktop-file-utils
+	test? ( dev-python/flake8[${PYTHON_USEDEP}] )"
```

Comment:
Is flake8 really necessary? We'd rather avoid depending on it given it won't find anything new compared to what upstream tests for already.

Reply:
Upstream may not check some or all release with flake8 in future.

Reply:
Let me rephrase. We really want you **not** to depend on flake8, as it's not going to gain py3.7 support anytime soon.

Reply:
That is a really much cleaner rephrasing :) Especially when it has a strict reason. Test phase removed.